pom.xml

```diff
@@ -125,27 +125,25 @@
         </configuration>
       </plugin>
       <plugin>
-	<groupId>org.codehaus.mojo</groupId>
```

Comment:
Let the record show there used to be a mix of TAB and spaces, which is why the whole block appears changed.